File: two sum/two_sum.py
class Solution:
    """
    @param: numbers: An array of Integer
    @param: target: target = numbers[index1] + numbers[index2]
    @return: [index1 + 1, index2 + 1] (index1 < index2)
    """
    def twoSum(self, numbers, target):
        # write your code here
        list1 = []
        for index in range(len(numbers)-1):
            i = 1
            while i < len(numbers) - index:
                if numbers[index] + numbers[index + i] == target:
                    list1.append(index + 1)
                    list1.append(index + i + 1)
                    return list1
                else:
                    i+=1

File: two sum/two_sum.cpp
class Solution {
public:
    /*
     * @param numbers: An array of Integer
     * @param target: target = numbers[index1] + numbers[index2]
     * @return: [index1 + 1, index2 + 1] (index1 < index2)
     */
    vector<int> twoSum(vector<int> &numbers, int target) {
        // write your code here
        for(int index = 0; index<numbers.size();index++){
            for(int i=1; i<numbers.size()-index ; i++){
                if(numbers[index]+numbers[index+i] == target){
                    vector<int> result(2);
                    result.push_back(index+1);
                    result.push_back(index+1+i);
                    return result;
                }
                else{
                    
                }
            }
        }
        
    }
};